pkg/sql/colexec/insert/insert.go

```diff
@@ -96,7 +97,21 @@ func Call(_ int, proc *process.Process, arg any) (bool, error) {
 
 		}
 	}
-	err := n.TargetTable.Write(ctx, bat)
+	txnOperator, err := proc.TxnClient.New()
```

Comment:
If it is the general Insert statement, the txn will be newed also?

Reply:
this txn only use in load stream